k8s/stubbornjava.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stubbornjava-deployment
+  labels:
+    app: sj-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sj-web
+  template:
+    metadata:
+      labels:
+        app: sj-web
+    spec:
+      containers:
+        - name: sj-web
+          image: containers.pkg.github.com/stubbornjava/stubbornjava-webapp:prod
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
```

Comment:
It's helpful to give names to your ports. This way you can define the actual port number in one place and just refer to it elsewhere by name.